render_test.go

```diff
@@ -223,13 +223,12 @@ func TestDiagnosticsOnArrowKeys(t *testing.T) {
 func TestDiagnosticsNilOnTextChange(t *testing.T) {
 	const selectAFrom = "select a from"
 	scenarios := []struct {
-		previousText     string
-		nextText         string
-		linesToTraceBack int
-		lastKey          Key
+		previousText string
+		nextText     string
+		lastKey      Key
 	}{
-		{previousText: selectAFrom, nextText: selectAFrom, linesToTraceBack: 0, lastKey: Enter},
-		{previousText: selectAFrom, nextText: "select a fro", linesToTraceBack: 0, lastKey: Enter},
+		{previousText: selectAFrom, nextText: selectAFrom, lastKey: Enter},
+		{previousText: selectAFrom, nextText: "select a fro", lastKey: Enter},
```

Comment:
I think I usually like to keep the scenarios similar between tests because I find it more readable. The test used it but they were constant, yeah. I removed this though so we don't need to change this back and forth again.